flink-python/pyflink/ml/tests/test_pipeline_it_case.py

```diff
@@ -169,3 +169,39 @@ def test_pipeline(self):
         # the first input is false since 0 + 0 is smaller than the max_sum 14.
         # the second input is true since 12 + 3 is bigger than the max_sum 14.
         self.assert_equals(actual, ["false", "true"])
+
+    def test_pipeline_from_and_to_java_json(self):
+
+        def get_java_pipeline_json():
+            wrapper = WrapperTransformer(selected_cols=["a", "b"])
+            wrapper._convert_params_to_java(wrapper._j_obj)
+            j_pipeline = get_gateway().jvm.org.apache.flink.ml.api.core.Pipeline()
+            j_pipeline.appendStage(wrapper._j_obj)
+            return j_pipeline.toJson()
```

Comment:
this piece of code replicates most of the logic in `Pipeline.to_json()` except for the hard-coded pipeline stage "select_cols". was wondering if there's a better way of making the test, e.g. use a json string literal here?

Reply:
A json string literal sounds good for me.